From 83aa072f712b68d4c74df95ed80d700bd5af93d8 Mon Sep 17 00:00:00 2001
From: Andrew Kelley <andrew@ziglang.org>
Date: Wed, 10 Sep 2025 02:08:11 -0700
Subject: [PATCH 112/133] langref: don't assume too much about pointer to
 packed struct field
X-Developer-Signature: v=1; a=openpgp-sha256; l=740; i=xtex@astrafall.org;
 h=from:subject; bh=6XOUdW2lJ77VLGrcdWp+MvSrEuhTSJLNQxvO8fnx0AQ=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn78njr6t/d1H82z/5Eqf7aoy+vtbytDPxrt3NBUPe30
 voI5S0VHRNZGMS4GCzFFFmKDBu8WXXS+UWXlcvCzGFlAhkiLdLAwMDAwMLAl5uYV2qkY6Rnqm2o
 Z2ioY6RjxMDFKQBTvesFI8Pe9o+fGB9YRn3eEcEnrTsl28XxznePnDSn6SUfdrCFGjxk+Kf9nKM
 i/c8zhjkTd29b+EVfl/1vZugrBjlnpe+vfT7G7ucFAA==
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit d7d50496d91a8d2e2c1dafa4e92231672b266677)
---
 doc/langref/test_overaligned_packed_struct.zig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/langref/test_overaligned_packed_struct.zig b/doc/langref/test_overaligned_packed_struct.zig
index b739b0d4079f..8bf52b32970b 100644
--- a/doc/langref/test_overaligned_packed_struct.zig
+++ b/doc/langref/test_overaligned_packed_struct.zig
@@ -8,7 +8,7 @@ const S = packed struct {
 test "overaligned pointer to packed struct" {
     var foo: S align(4) = .{ .a = 1, .b = 2 };
     const ptr: *align(4) S = &foo;
-    const ptr_to_b: *u32 = &ptr.b;
+    const ptr_to_b = &ptr.b;
     try expect(ptr_to_b.* == 2);
 }
 
-- 
2.52.0

